microsoft-edge/webview2/concepts/process-model.md

```diff
@@ -53,6 +53,9 @@ When the first `WebView2` instance is created for a given user data folder, the
 <!-- TODO: update with profile info -->
 The `CoreWebView2Environment` represents a user data folder and the collection of processes associated with it.  A given renderer process is not associated with a single `CoreWebView2` instance, because the renderer process can serve frames in multiple `CoreWebView2` instances that use the same user data folder, depending on website isolation.  See [Per-frame renderer processes - Site Isolation](https://developers.google.com/web/updates/2018/09/inside-browser-part1#site-isolation).
 
+On the other hand, if you create multiple `CoreWebView2Environment` objects configured the same way including sharing the same user data folder, they will all represent the same user data folder and the same associated collection of processes. Use any of them to create a `WebView2` with the one shared user data folder and associated collection of processes.
+
+If you try to create a `CoreWebView2Environment` using a user data folder which another `CoreWebView2Environment` is already using, and you don't configure the two `CoreWebView2Environment` objects the same, for example if they were created with different `CoreWebView2EnvironmentOptions.Language` values, then the second `CoreWebView2Environment` will fail to create `WebView2`s. This is true for `CoreWebView2Environment` objects that are in the same process or in different processes.
```

Comment:
@JasonAndrewWriter  - fyi.   also Acrolinx would suggest breaking up this long sentence.  If I tried that, I'd use VS Code as a scratchpad in this local .md file, then paste that rewrite into a Suggestion in this PR UI.
```suggestion
If you try to create a `CoreWebView2Environment` using a user data folder which another `CoreWebView2Environment` is already using, and you don't configure the two `CoreWebView2Environment` objects the same, for example if they were created with different `CoreWebView2EnvironmentOptions.Language` values, then the second `CoreWebView2Environment` will fail to create `WebView2` instances. This is true for `CoreWebView2Environment` objects that are in the same process or in different processes.
```